projects/cashmere/src/lib/sass/form-field.scss

```diff
@@ -68,14 +68,14 @@ $wrapper-padding-bottom: ($error-margin-top + $line-height) * $error-font-scale;
 
     background: $white;
 
-    @include formFieldFocusTransition();
+    //@include formFieldFocusTransition();
```

Comment:
Please delete instead of commenting out.

Reply:
This has been deleted.